src/deadline/client/cli/_deadline_cli.py

```diff
@@ -76,3 +78,6 @@ def main(ctx: click.Context, log_level: str):
 main.add_command(cli_job)
 main.add_command(cli_queue)
 main.add_command(cli_worker)
+
+if os.environ.get("ATTACHMENT_SYNC_JOB_USER") is not None:
```

Comment:
Only someone familiar with the feature name would know what this means. How about `JOB_ATTACHMENT_INPUT_OUTPUT_CLI`?

I'll name the manifest CLI `JOB_ATTACHMENT_MANIFEST_CLI` ?

Reply:
I don't think we need people to be familiar with this name or use this at this point. Worker integration is the only place should export the env var to use the CLIs. It's a staging feature flag likely not stay for long, once the feature is sign-off, we'll remove the condition altogether. A single feature name would be simpler.

Reply:
We will have customers that want to use / try this! Especially if this stays for more than 1 release (which it will) we should give it a good name.

Reply:
Synced offline, we agree to have 1 env var per delivery phase and use `JOB_ATTACHMENT_CLI`